### PR TITLE
 crefi to smallfile conversion , write_file to remote_file replacements , changing script according to tc description

### PR DESCRIPTION
**Pull request by yogesh-mane**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
crefi to smallfile conversion
replacing write_file function with remote_file function 
changing script as testcase does not require network disconnect (https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-11231)
This PR does not cover crefi scenario in cephfs_basic_tests.py (TBD)

### CODE REVIEW

**Comment by yogesh-mane** [comment]
rhel 8 baremetal logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1613995041586/
rhel 7 baremetal logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614062881424/
rhel 8 conatiner logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614015174826/
rhel 7 container logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614078949665

File: tests/cephfs/CEPH-10625_11225.py [hunk 1]
```diff
@@ -244,34 +244,22 @@ def run(ceph_cluster, **kw):
...
-                        cmd='sudo %s %s%s --fop %s -t %s '
-                            '--multi -b 10 -d 10 -n 10 -T 10 '
-                            '--random --min=1K --max=%dK' %
-                            ("python3 /home/cephuser/Crefi/crefi.py",
-                             client_info['mounting_dir'],
-                             dir_name, rand_ops, rand_filetype, rand_count),
-                        long_running=True)
+                        cmd='sudo python3 smallfile/smallfile_cli.py '
+                            '--operation %s --threads 10 --file-size 4'
+                            '--file-size 4 --files 1000 '
```
Comment (psathyan) [inline tests/cephfs/CEPH-10625_11225.py:257]: `--file-size 4` option is being provided twice... I think it is incorrect. At some point of time, you may want to start using format strings

`cmd = "sudo python3 smallfile/smallfile_cli_py --operation {ops[i]} --threads 10 --file-size 4 --files 1000 --files-per-dir 10 --dirs-per-dir 2 --top {client_info[mounting_dir]}{dir_name}`

File: tests/cephfs/cephfs_utils.py [hunk 2]
```diff
@@ -95,14 +95,31 @@ def get_clients(self, build):
...
                 output.split()
                 if 'attr' not in output:
                     node.exec_command(cmd='sudo yum install -y attr')
                 node.exec_command(cmd='sudo yum install -y gcc python3-devel',
                                   check_ec=False)
 
+                if 'xattr' not in output:
+                    if (build.endswith('7') or build.startswith('3')):
+                        node.exec_command(cmd="sudo subscription-manager repos --enable"
+                                              " rhel-7-server-optional-rpms --enable"
```
Comment (rakeshgm) [inline tests/cephfs/cephfs_utils.py:109]: we are already enabling repos in `install-prereq.py`. any reason why you need it here again.

File: tests/cephfs/cephfs_utils.py [hunk 3]
```diff
@@ -95,14 +95,31 @@ def get_clients(self, build):
...
+                                              " rhel-server-rhscl-7-rpms", long_running=True)
+                        node.exec_command(cmd="sudo yum -y install @development",
+                                          long_running=True)
+                        node.exec_command(cmd="sudo yum -y install rh-python36",
+                                          long_running=True)
+                        node.exec_command(cmd="sudo yum -y install rh-python36-numpy"
+                                              "rh-python36-scipy", long_running=True)
+                        node.exec_command(cmd="sudo yum -y install rh-python36-python-tools"
+                                              " rh-python36-python-six", long_running=True)
+                        node.exec_command(cmd="sudo yum install -y libffi libffi-devel")
```
Comment (rakeshgm) [inline tests/cephfs/cephfs_utils.py:119]: you could use a list here and iterate overs this, instead of calling this exec_command multiple times

**Comment by veera-raghava-reddy** [comment]
Hi @yogesh-mane,
cephci run is trunck in the following test for long time. 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614197869763/MDS-pinning-system_tests_0.log

2021-02-25 08:33:21,189 - ceph.ceph - INFO - Command completed successfully
2021-02-25 08:33:21,190 - tests.cephfs.cephfs_utils - INFO - Starting the network..

File: tests/cephfs/cephfs_utils.py (hunk 2 above)
Comment (yogesh-mane) [reply tests/cephfs/cephfs_utils.py:109]: I have tried using just install-prereq.py , but these repos are not getting enabled

**Comment by psathyan** [comment]
@yogesh-mane would it make sense to close this PR? I believe @AmarnatReddy has pushed most of the changes done in this PR in a new one.

**Comment by yogesh-mane** [comment]
>@yogesh-mane would it make sense to close this PR? I believe @AmarnatReddy has pushed most of the changes done in this PR in a new one.

Yes @psathyan , closing this PR

**Pull request closed without merging**